src/Cardano/Wallet.hs

```diff
@@ -360,8 +407,8 @@ mkWalletLayer db network = WalletLayer
         -> ExceptT ErrNoSuchWallet IO ()
     restoreBlocks wid blocks tip = do
         let (inf, sup) =
-                ( slotId . header . head $ blocks
-                , slotId . header . last $ blocks
+                ( view #slotId . header . head $ blocks
```

Comment:
is there a real need to use lens sintax here?

Reply:
Unfortunately yes, because we have conflicting record names. I am not quite satisfied with that, mostly because we don't use lens consistently at the moment. So, it feels really ad-hoc ine some cases.

Reply:
aha I see - I didn't noticed name conflict. In that case it looks sensible (not ideal - but still very clear to the reader)